UpdateCgnOrchestrator/__tests__/handler.test.ts

```diff
@@ -16,8 +16,9 @@ import {
   StatusEnum as RevokedCgnStatusEnum
 } from "../../generated/definitions/CardRevoked";
 import { ActivityResult } from "../../utils/activity";
+import { DEFAULT_EYCA_UPPER_BOUND_AGE } from "../../utils/config";
 import { MESSAGES } from "../../utils/messages";
-import { handler } from "../index";
+import { updateCgnOrchestratorHandler } from "../handler";
```

Comment:
Sorry, my suggestion was wrong. Should be capitalised handler name `UpdateCgnOrchestratorHandler` like [this](https://github.com/pagopa/io-functions-cgn/pull/40#discussion_r597542723)